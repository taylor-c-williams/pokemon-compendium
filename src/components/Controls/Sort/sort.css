.sort-body {
	background-image: url('../clouds.png');
	background-repeat: no-repeat;
	background-size: contain;
	background-position-x: right;
}
