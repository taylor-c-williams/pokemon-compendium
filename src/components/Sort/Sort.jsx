export default function Sort({ sortOrder, setSortOrder }) {
	return (
		<div>
			<select
				aria-label="sort"
				value={sortOrder}
				onChange={(e) => setSortOrder(e.target.value)}
			>
				<option value="asc">Ascending</option>
				<option value="desc">Descending</option>
			</select>
		</div>
	);
}
