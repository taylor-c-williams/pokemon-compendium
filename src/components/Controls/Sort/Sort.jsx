import './sort.css';

export default function Sort({ sortOrder, setSortOrder }) {
	return (
		<div className="sort-body">
			<section className="title">Sort By Pokedex Order:</section>
			<br />
			<select
				aria-label="sort"
				value={sortOrder}
				onChange={(e) => setSortOrder(e.target.value)}
			>
				<option value="asc">Ascending</option>
				<option value="desc">Descending</option>
			</select>
		</div>
	);
}
