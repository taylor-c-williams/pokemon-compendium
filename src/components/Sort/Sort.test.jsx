import { screen, render } from '@testing-library/react';
import Sort from './Sort';

it('renders the Sort component accurately', () => {
	render(<Sort />);
	const allSorts = screen.getAllByLabelText('sort');
	expect(allSorts).toMatchSnapshot();
});
