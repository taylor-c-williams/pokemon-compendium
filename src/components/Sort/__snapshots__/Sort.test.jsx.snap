// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Sort component accurately 1`] = `
Array [
  <select
    aria-label="sort"
  >
    <option
      value="asc"
    >
      Ascending
    </option>
    <option
      value="desc"
    >
      Descending
    </option>
  </select>,
]
`;
